### DISCUSSION TITLE
Label and icon for README section placement is off

### DISCUSSION THREAD

**Discussion by tooomm** [community/community#141306] [Mobile]
See screenshot, for other sections like the branch all is left aligned.
But the `README.md` label is right aligned, and the corresponding `(i)` icon is somewhere placed in the middle: 

<img src="https://github.com/user-attachments/assets/5b762e57-a8ea-4bbd-8c86-20f2ae4783bc" width="400">

<br>

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.181.0 (143346435)
Device: iPhone SE (2nd generation) (iOS 17.6.1)
</details>

**Comment by nthhanh526** [comment]
Tập vooc nokia chainlock

**Comment by tooomm** [comment]
Resolved for me on newest versions.

It would be great if the devs or product managers inform once they acknowledge/confirm certain reports and work on them or fixes are in place.


Edit:
It appears to only show correctly for repos that I have write access to and that show the 'Edit' button on the right to make changes to the README file.
The example repo in the screen above - and others without write access - still have this issue!

**Comment by mohamedayoub97** [comment]
It looks like you ran into an issue where the README.md label and the (i) icon weren’t aligned properly. Glad to hear it got fixed for you in the latest versions—especially for repos where you have write access and can edit the README.

However, it still seems like the problem pops up in repos where you don’t have write access. It would definitely be helpful if the devs or product managers could let us know when they’ve seen and confirmed issues like this, so we’re not left wondering if it’s being worked on.

If it keeps happening, you might want to report it to GitHub support or the issue tracker, so they can prioritize it!
